index.html

```diff
@@ -19,7 +19,9 @@
 
 		<div ng-app="app">
 			<div ng-controller="TestController as test">
-				<rl-rich-text-editor ng-model="test.text"></rl-rich-text-editor>
+				Selection: {{test.selection}}
+				<rl-select ng-model="test.selection" options="test.options" option-as="item"
```

Comment:
Ended up realizing that the messageLog templating has an issue with this currently... Since the ng-repeat uses an query expression, it doesn't currently support `message-as`. Didn't realize this was an issue at first because my templates must have been too simple. But looks like you have to assume that `$entry` will be available as the message for now. Can't see a way around it at the moment... :disappointed: